### PR TITLE
Bump pgxn to version 11.0.6

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: META.json [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
          "docfile": "README.md",
-         "version": "11.0.5"
+         "version": "11.0.6"
       }
    },
    "prereqs": {
       "runtime": {
          "requires": {
-            "PostgreSQL": "11.0.0"
+            "PostgreSQL": "13.0.0"
```
Comment (onurctirtir) [inline META.json:19]: maybe 14.5.0

**Review by onurctirtir — APPROVED** [review]

**Comment by onurctirtir** [comment]
oh wait

File: META.json (hunk 1 above)
Comment (onurctirtir) [inline META.json:19]:
```suggestion
            "PostgreSQL": "12.0.0"
```

**Review by onurctirtir — CHANGES_REQUESTED** [review]

**Comment by gurkanindibay** [comment]
> oh wait

waiting for you input

**Comment by onurctirtir** [comment]
> > oh wait
> 
> waiting for you input

sorry, everything is fine

**Review by onurctirtir — APPROVED** [review]

**Pull request merged**